doc/man3/SSL_export_keying_material.pod

```diff
@@ -13,14 +15,29 @@ SSL_export_keying_material - obtain keying material for application use
                                 const unsigned char *context,
                                 size_t contextlen, int use_context);
 
+ int SSL_export_keying_material_early(SSL *s, unsigned char *out, size_t olen,
+                                      const char *label, size_t llen,
+                                      const unsigned char *context,
+                                      size_t contextlen, int use_context);
```

Comment:
The `use_context` flag is a remnant of the pre-TLS-1.3 API for this feature where no context and empty context were distinct inputs. In TLS 1.3, we changed it so that the two are considered identical (no one ever bothered with the distinction). It might make sense to remove that parameter to match.

https://tlswg.github.io/tls13-spec/draft-ietf-tls-tls13.html#rfc.section.7.5
https://github.com/tlswg/tls13-spec/issues/546

Reply:
Good point.  Will do.

Reply:
3f45aa4 removed use_context argument.